### PR TITLE
ladok_uid is now included as integration_id in output csv

### PR DESCRIPTION
**Pull request by kstarzecki**

### CODE REVIEW

**Comment by exacs** [comment]
The code looks nice! Can you add also the same Ladok UID as integration_id for sections?

**Comment by Delodax** [comment]
Fixed some "standard" issues.

File: server/createCoursesFile.js [hunk 1]
```diff
@@ -28,6 +29,7 @@ function createCourseOfferingObj (courseOffering) {
     tutoringLanguage: courseOffering.language,
     departmentCode: courseOffering.department_code,
     schoolCode: courseOffering.school_code,
+    integrationId: courseOffering.course_round_applications[0].ladok_uid,
```
Comment (Delodax) [inline server/createCoursesFile.js]: Gonna look into this LOC. It is a bit unclear to me if we need to handle the situation where a course offering has multiple applications.

Talked to @murkchand about it and she was gonna forward some inquiries to Jessica!

Comment (kstarzecki) [reply server/createCoursesFile.js]: Thanks, initially, I wanted to ask about that too, but I checked what is returned for anything other than course_round_applications[0], and I didn't get anything, so I left it out for the review.

Very curious to find what's the feedback!

Comment (Delodax) [reply server/createCoursesFile.js]: @kstarzecki / @exacs : So the information that @murkchand got from Jessica was that right now we have one-to-one match between applications and offerings. Thus we can safely simply fetch the first element of the `course_round_applications` and be done with it :)

The reason for there being an array was that historically there could be multiples. Also, there was a potential that in the future, this concept would return! But as long as Team e-lärande and StudAdm makes sure to communicate such API changes, we should be able to fix it then and there 😼

**Pull request merged**